scripts/ci/check_sched_scalable.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -e
+
+if [ "${CC#clang}" != "${CC}" ] ; then
+	export CXX="clang++"
+fi
+
+cd "$(dirname "$0")"/../..
+./bootstrap
+./configure
+make -j $(nproc)
+
+echo 1000 | tee /proc/sys/vm/nr_hugepages
+mkdir -p /mnt/huge
+mount -t hugetlbfs nodev /mnt/huge
+
+export CI="true"
+
+ODP_SCHEDULER=scalable make check
```

Comment:
no need to separate script file for each run. Just pass new environment variable to Docker. -e ODP_SCHEDULER="${ODP_SCHEDULER}" and in .travis.yaml env: setup it, like "env:  ODP_SCHEDULER=scalable"

Reply:
Script files removed. Used -e ODP_SCHEDULER=sp. Didn't add to matrix (only make check with gcc).

Reply:
@psavol script files are still here.

Reply:
Will be removed in next version...